### PR TITLE
Improve superset list view content layout

### PR DESCRIPTION
**Pull request by graceguo-supercat**
superset has a few pages that were using  default template provided by AppBuilder framework.
old_slices

One problem with this layout is that title, filter, pagination are all stacked row by row, which took a lot of space before actual list content. So we want to make a new list view layout, to make top section cleaner and keep all the existed functionalities.
new_list_view

### CODE REVIEW

File: superset/templates/appbuilder/general/widgets/search.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    $('.list-search-container').on('hidden.bs.dropdown', function () {
+        checkSearchButton();
+    });
+    $(document).ready(function() {
+        checkSearchButton();
+    });
+
+	(function($) {
```
Comment (xrmx) [inline superset/templates/appbuilder/general/widgets/search.html]: tabs instead of spaces?

Comment (graceguo-supercat) [inline superset/templates/appbuilder/general/widgets/search.html]: not sure which line you mean to use tabs. But it seems all superset code base used spaces not tabs for line indentation.

Comment (xrmx) [inline superset/templates/appbuilder/general/widgets/search.html]: I mean you are using tabs instead of spaces :)

Comment (graceguo-supercat) [inline superset/templates/appbuilder/general/widgets/search.html]: Thanks! fixed.

File: superset/templates/appbuilder/general/widgets/search.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        </div>
+    </form>
+</div>
+
+<script>
+(function($) {
+    function checkSearchButton() {
+        var hasFilter = $('.filters tr').length;
+        if (hasFilter) {
+        	$('.filters a.remove-filter').off('click', checkSearchButton);
```
Comment (xrmx) [inline superset/templates/appbuilder/general/widgets/search.html]: nit: still one tab here

Comment (graceguo-supercat) [inline superset/templates/appbuilder/general/widgets/search.html]: :-( another fix.

**Comment by mistercrunch** [comment]
This PR doesn't look right... 76 files changed?

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**